app/app.h

```diff
@@ -3,5 +3,39 @@
 
 namespace vsite::oop::v3 
 {
+	struct student {
+		std::string name;
+		int grade = 0;
+	};
 
-}
\ No newline at end of file
+	class results {
+		int count = 0;
+		int pos = 0;
+		student* studNiz = nullptr;
+
+	public:
+		results(int count = 0);
+		void add(student s);
+		unsigned has_grade(int g);
+		unsigned starts_with_letter(char c);
+		~results();
+	};
+
+	class array {
+		unsigned _size;
+		double* nizArr;
+
+	public:
+		array(unsigned s = 0, double value = 0.0);
+		array(const array& source);
+		array(array&& source) noexcept;
+
```

Comment:
Nema destruktora (definitivno je potreban)?

Reply:
Nisam bio na zadnjem predavanju te nisam siguran kada bi se točno destruktori trebali koristit.